mlir/lib/Target/CppOutput/gridwise_convolution_implicit_gemm_v4r4.cpp

```diff
@@ -1096,19 +1213,15 @@ std::unique_ptr<llvm::StringRef> mlir::translateModuleToMIOpenCFlags(ModuleOp m)
       PopulateParams populateParams;
       populateParams.paramsFromCtx(convContext, parameters);
 
-      // parameters fixed.
-      parameters["CK_PARAM_TUNABLE_GEMM_M_LEVEL0_CLUSTER"] = 4;
-      parameters["CK_PARAM_TUNABLE_GEMM_N_LEVEL0_CLUSTER"] = 4;
-      parameters["CK_PARAM_TUNABLE_GEMM_M_LEVEL1_CLUSTER"] = 4;
-      parameters["CK_PARAM_TUNABLE_GEMM_N_LEVEL1_CLUSTER"] = 4;
-
       // paeterrameters varies depend on layout but set as fixed now
       parameters
           ["CK_PARAM_TUNABLE_GEMM_C_THREAD_COPY_DST_DATA_PER_WRITE_GEMM_N1"] =
               1;
 
       TunableParameters params(parameters);
 
+      //TO-DO: need a condition that outputVecLen % ClusterLengthN1 
+#if 0
```

Comment:
Can you elaborate why macro the below tuning parameter logic on gemmC off? We need a better way to do it than `#if 0`.